graylog_hook_test.go

```diff
@@ -283,12 +285,26 @@ func TestStackTracer(t *testing.T) {
 		t.Error("Stack Trace is not a string")
 	}
 	stacktraceRE := regexp.MustCompile(`^
-.+/logrus-graylog-hook(%2ev2)?.TestStackTracer
+(.+)?logrus-graylog-hook(%2ev2)?.TestStackTracer
```

Comment:
this change was made because when running tests locally this line was started directly form logrus-graylog-hook without the /